tests/smoke/smoke_test.go

```diff
@@ -108,6 +108,12 @@ var _ = Describe("Smoke Tests", func() {
 				cfLogs := cf.Cf("logs", appName, "--recent")
 				return string(cfLogs.Wait().Out.Contents())
 			}, 2*time.Minute, 2*time.Second).Should(ContainSubstring("Hello World from index"))
+
+			By("verifying that the application's metrics are available.")
+			Eventually(func() string {
+				cfApp := cf.Cf("app", appName)
+				return string(cfApp.Wait().Out.Contents())
+			}).Should(ContainSubstring("cpu"))
```

Comment:
Won't the string "cpu" always present, regardless of whether metrics are being reported?

Reply:
Update: We don't think it is worth blocking the PR merge on this, but we're planning to remove this assertion after merging. The smoke tests are very basic (just testing you can push, curl, and get logs for an app), and this assertion feels a little out of place in that. Plus, it seems like it wouldn't likely catch any problems because of what @Birdrock mentioned.